README.md

```diff
@@ -239,7 +239,7 @@ Example with custom libvirt management network:
 
 ```yaml
 static:
-  box: centos8
+  box: centos9
```

Comment:
I didn't comment on every instance of `centos9`, but please review them for where they need to be `centos9-stream.